pkgs/os-specific/linux/kernel-headers/default.nix

```diff
@@ -29,13 +29,17 @@ let
     destination = "/include/byteswap.h";
   };
 
+  linuxArchMapping = {
+    "loongarch64" = "loongarch";
```

Comment:
This should be set in `linuxArch` in lib/systems/default.nix.

Reply:
Actually, it is weird, when I was build it's actually facing this kind of issue, the `linux-headers` need is different with the `linuxArch`, but it seems to work fine right now.